src/open_inwoner/configurations/bootstrap/models.py

```diff
@@ -0,0 +1,124 @@
+from django.core.exceptions import FieldDoesNotExist
+
+from open_inwoner.conf.utils import config
+from open_inwoner.configurations.models import SiteConfiguration
+
+
+class ConfigValue:
+    """
+    Descriptor for retrieving configuration settings from environment variables
+    """
+
+    def __init__(self, namespace, verbose_name, help_text, name=None):
+        self.namespace = namespace
+        self.verbose_name = verbose_name
+        self.help_text = help_text
+        self.name = name
+
+    def __set_name__(self, owner, name):
+        self.public_name = name
+        self.private_name = "_" + name
+        self.setting_name = f"{self.namespace}_" + name.upper()
+
+    def __get__(self, instance, objtype=None):
+        return config(self.get_setting_name(), default=None)
+
+    def __set__(self, instance, value):
+        pass
+
+    def get_setting_name(self) -> str:
+        if setting_name := getattr(self, "setting_name", None):
+            return setting_name
+        return f"{self.namespace}_" + self.name.upper()
+
+
+class ConfigMeta(type):
+    """
+    Meta class for populating configuration classes with descriptors
+    """
+
+    def __init__(cls, name, bases, dct):
+        model_fields = (
+            field
+            for field in vars(cls.model)
+            if not field.startswith("__") and field not in cls.excluded_fields
+        )
```

Comment:
This should filter on the Field base class (the way the `_iter_descriptors` does) (there could be more members without an underscore then just fields)